plugins/jenkins/api-report.md

```diff
@@ -118,8 +137,29 @@ export const LatestRunCard: (props: {
 // @public (undocumented)
 export const LEGACY_JENKINS_ANNOTATION = 'jenkins.io/github-folder';
 
+// Warning: (ae-missing-release-tag) "Project" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Please avoid introducing more warnings to the API report, this one is avoid by marking `Project` as `@public`